octoprint_slic3r/__init__.py

```diff
@@ -166,7 +166,7 @@ def get_slicer_properties(self):
 	def get_slicer_default_profile(self):
 		path = self._settings.get(["default_profile"])
 		if not path:
-			path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "profiles", "default.profile.ini")
+			path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "static/profiles", "default.profile.ini")
```

Comment:
Ok, I think I see what you mean. In this case, static and templates are folders used for code files, where static has *.js files and templates *.jinja2 files that generate the HTML for the user interface. Profiles folder contains settings to be used in case that there is not any profile uploaded, which would be located at (I have to check this because I am telling by heart) .octoprint/slicingProfiles/scli3r